src/components/Button/Button.js

```diff
@@ -0,0 +1,34 @@
+import { Link } from "react-router-dom";
+import propTypes from "prop-types";
```

Comment:
I'm not sure how many times I need to say the same thing. Be consistent - `PropTypes`.